src/zmq/zmqnotificationinterface.cpp

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) 2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "zmqnotificationinterface.h"
+#include "zmqpublishnotifier.h"
+
+#include "version.h"
+#include "main.h"
+#include "streams.h"
+#include "util.h"
+#include "netbase.h"
```

Comment:
nit: i think this include can be removed. Compiles fine on OSX/Debian without this include.